java/lang/security/jackson-unsafe-deserialization.yaml

```diff
@@ -22,6 +22,22 @@ rules:
                   metavariable: $TYPE
                   regex: (Object|Serializable|Comparable)
               - pattern: $OM.readValue($JSON, $CLASS.class);
+          - patterns:
+              - pattern-inside: |
+                  class $CLASS {
+                    ObjectMapper $OM;
+                    $INITMETHODTYPE $INITMETHOD(...) {
+                      ...
+                      $OM = new ObjectMapper();

```

Comment:
include `...` between these two lines, otherwise if there's a variable defined between these two lines, it would not match

Reply:
Yes, I also agree that's better. Please go ahead and push the change! Thanks!